src/browser/navigation-panel.js

```diff
@@ -13,18 +13,24 @@ const {Element} = require('./element');
 const {navigateTo, showTabStrip, focus} = require('./actions');
 const {KeyBindings} = require('./keyboard');
 const url = require('./util/url');
+const {readTheme} = require('./theme');
 
-const WindowControls = Component(options =>
-  DOM.div({className: 'windowctrls'}, [
+const WindowControls = Component(({webViewer}) => {
+  const theme = readTheme(webViewer);
+
+  return DOM.div({className: 'windowctrls'}, [
     DOM.div({className: 'windowctrl win-close-button',
+             style: {backgroundColor: theme.close},
```

Comment:
Are you sure we want that? Some of these styles are shared between multiple components (e.g. `theme.buttons`). I can do that if you really think its the right way forward.